index.html

```diff
@@ -32,12 +32,15 @@
                 <a href="">WELCOME</a>
                 <nav class="toggle">
                     <ul class="nav-list">
-                        <li> <a href="#">Portfolio</a> </li>
-                        <li> <a href="#">About</a> </li>
-                        <li> <a href="#">Contact</a> </li>
+                        <li class="link"> <a href="#portfolio">Portfolio</a></li>
+                        <li class="link"> <a href="#About">About</a></li>
+                        <li class="link"> <a href="#contact">Contact</a> </li>
                     </ul>
-                    <button class="menu"><i class="fa-solid fa-bars"></i></button>
                 </nav>
+                <div class="hamburger">
+                    <button class="close"><i class="fa-solid fa-xmark"></i></button>
+                </div>
+                <button class="menu"><i class="fa-solid fa-bars"></i></button>
```

Comment:
- [ ] Please, try to fix the navbar view on the desktop and mobile view also. You can see, that on the desktop version, there are no nav links:
<img width="1145" alt="Screen Shot 2022-03-08 at 19 11 08" src="https://user-images.githubusercontent.com/49064106/157266303-b6a4f69a-cfa7-499e-a4b5-c46c11469f6d.png">

- [ ] Your mobile menu is not cover the full page. You can also see the logo on top of the mobile menu. Considering the Figma template the mobile menu should not consist of the logo and should cover the full page:
![2022-03-08 19 12 49](https://user-images.githubusercontent.com/49064106/157267049-43530a69-4697-4f42-b826-fbf78f2b6011.gif)

> Here is the more correct version of the menu displaying:

![2022-03-08 19 16 24](https://user-images.githubusercontent.com/49064106/157267456-bbd0e0da-1687-497d-ae24-b115561d8f09.gif)

> Here is also the Figma template:
<img width="185" alt="Screen Shot 2022-03-08 at 19 17 25" src="https://user-images.githubusercontent.com/49064106/157267673-3101586f-661d-4c82-a188-30815d4bece7.png">

- [ ] Additionally, please, consider fixing the `@media only screen and (min-width: 992px) ` - 992px should be the board for switching from desktop to mobile version and the hamburger icon should appear. In your version it does not happen on 992px, but happened on 1050px:
![2022-03-08 19 23 27](https://user-images.githubusercontent.com/49064106/157268846-f0a42b55-f68b-489b-a254-092161f8cc7a.gif)

Reply:
Hello @elmar8287, as requested by you. I have:
1.  set the JS linter,
2.  Fixed the navbar view on the desktop and mobile view also,
3.  Made the mobile menu to cover the full page,
4.  Fixed the @media only screen and (min-width: 992px). Kindly check to confirm.
   
    Thank you